parler/models.py

```diff
@@ -466,13 +475,14 @@ def _get_translated_model(self, language_code=None, use_fallback=False, auto_cre
             if not self._state.adding or self.pk is not None:
                 _cache_translation_needs_fallback(self, language_code, related_name=meta.rel_name)
 
-        if lang_dict['fallback'] != language_code and use_fallback:
+        if language_code not in lang_dict['fallbacks'] and use_fallback:
             # Jump to fallback language, return directly.
             # Don't cache under this language_code
-            try:
-                return self._get_translated_model(lang_dict['fallback'], use_fallback=False, auto_create=auto_create, meta=meta)
-            except meta.model.DoesNotExist:
-                fallback_msg = " (tried fallback {0})".format(lang_dict['fallback'])
+            for fallback_lang in lang_dict['fallbacks']:
+                try:
+                    return self._get_translated_model(fallback_lang, use_fallback=False, auto_create=auto_create, meta=meta)
+                except meta.model.DoesNotExist:
+                    fallback_msg = " (tried fallbacks {0})".format(', '.join(lang_dict['fallbacks']))
```

Comment:
I think this can be made a bit cleaner, and more precise. e.g. only set this message when all fallbacks failed.